### PR TITLE
learning_rates per boosting round for CV too

### PR DESCRIPTION
**Pull request by simonnanty**
Adding possibility to have adaptive learning rate in xgboost.cv as in xgboost.train

### CODE REVIEW

**Comment by tqchen** [comment]
We would prefer not add additional arguments for the cv, and trains, instead switching these to callback mechanism. So I will close this for now.
